### PR TITLE
css/tooltip-text

### LINKED ISSUE #1875: convert css for tooltip

### LINKED ISSUE #1877: convert css for textinput

### LINKED ISSUE #1878: convert css for textarea

### LINKED ISSUE #1879: convert css for text

### PR DESCRIPTION
**Pull request by EdwardIrby**
resolve #1875 
resolve #1877 
resolve #1878 
resolve #1879

### CODE REVIEW

File: packages/text/src/react/__specs__/heading.spec.tsx [hunk 1]
```diff
@@ -4,13 +4,26 @@ import { axe } from 'jest-axe'
...
 import * as stories from '../__stories__/heading.story'
+import { Heading } from '../index'
 
 describe('Text', () => {
   describe('Heading', () => {
     const cases = convertStoriesToJestCases(stories)
 
     it.todo('forwards the ref')
 
+    it('composes className', () => {
```
Comment (jaketrent) [inline packages/text/src/react/__specs__/heading.spec.tsx:15]: With the addition of this an other specs, I think the "Class Override" stories could safely be removed.

**Review by jaketrent — CHANGES_REQUESTED** [review]
Found several things that could use an update.

File: packages/textinput/src/react/index.tsx [hunk 2]
```diff
@@ -186,19 +172,28 @@ const TextInput = forwardRefWithStatics<
...
 
           {icon && (
-            <div {...styles.icon({ appearance, icon, iconAlign, themeName })}>
+            <div
+              className={styles.icon({
+                appearance,
+                icon,
+                iconAlign,
+                themeName
+              })}
```
[the remark this answers is not in the document]
Comment (EdwardIrby) [reply packages/textinput/src/react/index.tsx:178]: I'll try to do that going forward.

File: packages/text/src/css/index.css [hunk 3]
```diff
@@ -0,0 +1,186 @@
...
+.psds-text__label--color-secondary.psds-theme--light {
+  color: var(--ps-colors-text-icon-low-on-light);
+}
+.psds-text__label--color-primary.psds-theme--dark {
+  color: var(--ps-colors-text-icon-high-on-dark);
+}
+.psds-text__label--color-secondary.psds-theme--dark {
+  color: var(--ps-colors-text-icon-low-on-dark);
+}
+.psds-text__label--size-xSmall {
```
[the remark this answers is not in the document]
Comment (EdwardIrby) [reply packages/text/src/css/index.css]: I just realized we have a dashify utility. I've been converting these manually in the style sheets and vars. Let me know what you think of the change to vars on tooltip and text

**Review by sweeting — CHANGES_REQUESTED** [review]
Tooltip, TextInput, Textarea look good.

Text: I didn't see size stories for label and the only size of label I see was wrapped in an `<h1>` and font-size is 32px.

Specs for the text label components can be found in [this Figma file](https://www.figma.com/file/YsTklBecdddy9RZ3HXddIseD/PSDS-Web-Components?node-id=11732%3A50314).

image

**Comment by EdwardIrby** [comment]
> this Figma file

I'll go ahead and see what's up with these. Thanks!

**Comment by jaketrent** [comment]
Not sure if it'll make a diff here or not, but I saw that master was back merged. I think we should avoid that. This is a branch to merge into convertScript. I would advise only back-merging convertScript at this point.

**Pull request merged**